woocommerce/woocommerce-ios
# Update package actions and repository

## Description


- Minor adaptations for the `WooShippingPackagesRepository` to prepare for connecting the backend response data.
- Move saving template backend request to repository

## Testing information


1. Install and set up the Woo Shipping extension on your store.
2. Build and run the app with the `revampedShippingLabelCreation` feature flag enabled.
3. Create an order with the processing status and at least one physical product.
4. In the order details, select "Create Shipping Label."
5. Tap "Select a Package" button
6. Input values for custom package
7. Toggle switch to save template
8. Save package as template
9. Check on web that the new template with data you added is saved

## Screenshots


![Simulator Screen Recording - iPhone 16 Pro - 2024-11-21 at 12 28 26](https://github.com/user-attachments/assets/9b05c118-2745-4f44-8529-f2072429f1f0)

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
